portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk.mdx

```diff
@@ -61,27 +61,27 @@ Click [here](/learn/connect-to-blockchain) to learn how to connect to the blockc
 
 For this guide, you will need to instantiate the SDK following [these](/learn/connect-to-blockchain#using-your-own-signer-with-thirdweb) steps, using your own signer with thirdweb.
 
-### 3. Define which module to use
+### 3. Define which contract to use
 
-Now we need to define which module we want to use. This is the module we just created inside our project Minting NFTs with TypeScript.
-We called the module '**Marketplace written in TypeScript**'.
-This module has an address. We need to pass the address here. You can find the address here, under '**Marketplace written in TypeScript**':
+Now we need to define which contract we want to use. This is the contract we just created inside our dashboard.
+We called the contract '**Marketplace written in TypeScript**'.
+This contract has an address. We need to pass the address here. You can find the address here, under '**Marketplace written in TypeScript**':
 
-![](/assets/portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk/20-module.png)
+![](/assets/portal/guides/create-your-own-auction-with-thirdweb-typescript-sdk/20-contract.png)
```

Comment:
this picture doesn't load in the guide, did you modify the image name?

Reply:
updated the name